fsociety/core/errors.py

```diff
@@ -0,0 +1,24 @@
+from typing import Optional, Union
+
+from rich.style import Style
+
+from fsociety.console import console
+from fsociety.core.config import get_config
+
+# Config
+config = get_config()
+
+
+def errorhandle(error: Exception, style: Optional[Union[str, Style]] = "red") -> None:
+    console.print(str(error), style=style)
+    if config.getboolean("fsociety", "development"):
+        console.print_exception()

```

Comment:
Maybe we send the command to debug? Instead of just ignoring it.

Reply:
we're still printing the error itself, it's only the full stack that's not getting printed now. before it would print out as 
```log
Error: SomethingException {more message}

STACK
|
|
|
|
|
|
|
|
|
|

Error: SomethingException {more message}
```

Reply:
there's also no "sending" it to debug, it's going to be an if check here either way, the Console interface doesn't provide log levels.

I'm happy to differentiate it if desired, but i think that'll require adding on a bit more to the Console interface